django/db/models/sql/query.py

```diff
@@ -2052,6 +2052,24 @@ def extra_select(self):
         else:
             return self.extra
 
+    def is_alias_available_join(self, all_paths, alias):
+        """
+        Check alias for whether both from and to in all_paths.
+        If so shouldn't be removed
+        alias_refcount might be changed hence need joined_alias
+        to check available from join
+        """
```

Comment:
This docstring needs rewording. It should focus on what the function is doing and not on what the called should do.